e2e/transform/babel-jest/package.json

```diff
@@ -5,10 +5,10 @@
   "jest": {
     "collectCoverageOnlyFrom": {
       "<rootDir>/this-directory-is-covered/Covered.js": true,
```

Comment:
is this left with capital case on purpose?

Reply:
I did miss these files, sorry! I also deleted the left over files named `.e2e/transform/babel-jest/this-directory-is-covered/ExcludedFromCoverage.js`  and `./e2e/transform/no-babel-jest/this-directory-is-covered/ExcludedFromCoverage.js` in the same location